config/data/voc12.yaml

```diff
@@ -16,26 +16,28 @@ data:
     pattern:
       image: ~
       label: ~
-  id_mapping: ['background', 'aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']
-  pallete:
-    - [0, 0, 0]
-    - [128, 0, 0]
-    - [0, 128, 0]
-    - [128, 128, 0]
-    - [0, 0, 128]
-    - [128, 0, 128]
-    - [0, 128, 128]
-    - [128, 128, 128]
-    - [64, 0, 0]
-    - [192, 0, 0]
-    - [64, 128, 0]
-    - [192, 128, 0]
-    - [64, 0, 128]
-    - [192, 0, 128]
-    - [64, 128, 128]
-    - [192, 128, 128]
-    - [0, 64, 0]
-    - [128, 64, 0]
-    - [0, 192, 0]
-    - [128, 192, 0]
-    - [0, 64, 128]
\ No newline at end of file
+  label_image_mode: rgb
```

Comment:
I think it would be better to keep in upper case letters.

Reply:
I wrote it with lowercase on purpose to show that it allows lowercase.

But if it would be better if eqaulizing with uppercase, I'll fix it.